test/Templates.Test/Templates.Test.csproj

```diff
@@ -2,10 +2,12 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
```

Comment:
What is this used for? Was it meant to be included in the PR?

Reply:
Yep. This is to avoid a dependency on System.Security.Cryptography.Cng

Reply:
I guess I'm missing where it's used then. Is it a magic value that something in the compilation chain is looking for (and if so, what)?

Reply:
It's a define for an #if XPLAT that we use on the shared sources library that we bring in to setup an HTTPS certificate for the tests

Reply:
OK thanks!